app/assets/javascripts/admin/addon/components/admin-user-field-item.js

```diff
@@ -1,146 +1,137 @@
-import Component from "@ember/component";
+import Component from "@glimmer/component";
```

Comment:
Again if the editing was split into a different route and the form itself was a separate component, this could be wayyyy lighter. But that can be done in future :)